README.md

```diff
@@ -56,7 +49,197 @@ class AppServiceProvider extends ServiceProvider
 }
 ```
 
-## Updating Configuration
+## Vite Instructions
+If your app uses Webpack (Laravel Mix), skip to the [Webpack Instructions](#webpack-instructions).
+
+### Enabling SSR
+
+If you just installed a fresh Laravel Jetstream app with the --ssr flag, you can skip this step.
+
+Make sure that `inertia/laravel-inertia` is at least version `0.5.1`.
```

Comment:
I think this should be `inertiajs/inertia-laravel`? :)

Reply:
also... shouldn't the version constraint now be `^0.6.3`? I can't even install as-is.